dnf/repo.py

```diff
@@ -517,6 +485,10 @@ class Repo(dnf.conf.RepoConf):
     def __init__(self, name=None, parent_conf=None):
         # :api
         super(Repo, self).__init__(section=name, parent=parent_conf)
+        # <<< TEST
```

Comment:
It would be better to properly comment this section.
It's reason is not clear at the first sight.
Or maybe just explain what the "TEST" tests.

Reply:
Yeah, they're only clear to me :) They were not meant for the final PR, of course. But I'll fix them now, no problem. Thanks!